src/lib/dom.generated.d.ts

```diff
@@ -7014,14 +7076,12 @@ interface ImageData {
     width: number;
 }
 
-interface ImageDataConstructor {
+declare var ImageData: {

```

Comment:
This is due to a change in the script to unify the emitted constructor format, so that it is easier to add more constructor signatures to a type using external json data. This change in particular syncs with the version in the script repo.